cli/src/commands/fee-profile/deleteVideo.ts

```diff
@@ -0,0 +1,57 @@
+import FeeProfileCommandBase from '../../base/FeeProfileCommandBase'
+import { flags } from '@oclif/command'
+import { createType } from '@joystream/types'
+import chalk from 'chalk'
+import { formatBalance } from '@polkadot/util'
+
+const DEFAULT_ASSETS_NUM = 2
+
+export default class FeeProfileDeleteVideo extends FeeProfileCommandBase {
+  static description = 'Create fee profile of forum.delete_video extrinsic.'
+
+  static flags = {
+    assetsNum: flags.integer({
+      char: 'a',
+      default: DEFAULT_ASSETS_NUM,
+      description: 'Number of assets to use for estimating the costs/returns',
+    }),
+    storageBucketsNum: flags.integer({
+      char: 'S',
+      description:
+        'Number of storage buckets to use for estimating tx fee.\n' +
+        "By default this number will be based on the current chain's dynamic bag policy for channel bags",
+    }),
+    ...super.flags,
+  }
+
+  async run(): Promise<void> {
+    const api = await this.getOriginalApi()
+    const { assetsNum, storageBucketsNum: forcedStorageBucketsNum } = this.parse(FeeProfileDeleteVideo).flags
+    const dataObjectBloatBond = await api.query.storage.dataObjectStateBloatBondValue()
+    const videoBloatBond = await api.query.content.videoStateBloatBondValue()
+
+    const channelBagPolicy = await api.query.storage.dynamicBagCreationPolicies('Channel')
+    const storageBucketsNum = forcedStorageBucketsNum || channelBagPolicy.numberOfStorageBuckets.toNumber()
+
+    this.log(`Data object bloat bond: ${chalk.cyanBright(formatBalance(dataObjectBloatBond))}`)
+    this.log(`Video bloat bond: ${chalk.cyanBright(formatBalance(videoBloatBond))}`)
+    this.log('Parameters:')
+    this.jsonPrettyPrint(
+      JSON.stringify({
+        storageBucketsNum,
+        assetsNum,
+      })
+    )
+
+    const tx = api.tx.content.deleteVideo({ Member: 0 }, 0, assetsNum, createType('Option<u32>', storageBucketsNum))
+    const txFee = await this.getApi().estimateFee(this.pairs.alice, tx)
+    const costs = {
+      txFee,
+    }
+    const returns = {
+      dataObjectsBloatBond: dataObjectBloatBond.muln(assetsNum),
+      videoBloatBond,
```

Comment:
Wouldn't using boat bond values directly from chain state can show incorrect `JOY` returns, since the bloat bond values can change after the video was created?

Reply:
This command does not estimate the cost of deleting a given, existing video, but just _some_ video.
The intention behind those commands is that they can be used to verify how given chain parameters (like _WeightToFee_ function, bloat bond values etc.) affect the costs of some common extrinsics.
I can make the bloat bond confiugrable via `--bloatBond` flag, but I'm not sure if that's necessary given this purpose (so assuming the commands will mostly be ran against the local/dev chain anyway, where the user has full control over bloat bond values).

Reply:
Yeah, this implementation makes sense if commands are being used during local development